sale_stock_picking_blocking/views/sale_stock_picking_blocking_reason_view.xml

```diff
@@ -4,14 +4,14 @@
 <!--License
 AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).-->
 <odoo>
-    <record id="sale_stock_picking_blocking_reason_tree_view" model="ir.ui.view">
-        <field name="name">sale.delivery.block.reason.tree</field>
+    <record id="sale_stock_picking_blocking_reason_list_view" model="ir.ui.view">
```

Comment:
Updating the XML-ID is not a good idea IMO, this will force a lot of XML-IDs mapping during migration if we do that in all modules.

Is there any guidance / best practice regarding this? cc @OCA/crm-sales-marketing-maintainers